### DISCUSSION TITLE
How to use multiple host headers in the same request?

### DISCUSSION THREAD

**Discussion by alins1r** [projectdiscovery/nuclei#905] [Q&A] [answered]
**Describe the bug**
we can't set **two** host header to send in the requests

**Nuclei version**
Current Version: 2.3.8

**Screenshot of the error or bug**
![image](https://user-images.githubusercontent.com/67107893/128220508-268d84d9-b5aa-4101-b2a5-6f6b5f58a35a.png)

**template**

```
id: test-two-header

info:
    author: AmirMohammad Safari
    name: Testing Two Header
    severity: info

requests:
  - raw:
        - |
            GET / HTTP/1.1
            Host: {{Hostname}}
            Host: test.com
            Connection: close
            User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko)
            Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8
            Accept-Language: en-US,en;q=0.9

    matchers:
      - type: word
        words:
          - "test.com"
```

**Comment by Mzack9999** [comment, marked as answer]
In case you want to send non-RFC compliant HTTP requests you need to use `unsafe: true` (reference https://nuclei.projectdiscovery.io/templating-guide/protocols/http/#unsafe-http-requests)

**Comment by ehsandeep** [comment]
@alins1r also consider updating nuclei to the latest version, i.e [v2.4.2](https://github.com/projectdiscovery/nuclei/releases/tag/v2.4.2)
